### ISSUE TITLE
Setting Metadata Sync Schedule

### ISSUE THREAD

**Issue by luqmansyauqi** [metabase/metabase#4262]
Is there any way to set up metadata sync schedule to all or particular databases? We need to restrict the sync or move the schedule (for example in midnight) so it will not impact the performance (while everyone is using it).

Please advise!

**Comment by DannyDouglass** [comment]
This would be very helpful!

**Comment by luqmansyauqi** [comment]
@DannyDouglass for workaround I think we can create our own build by modifying this code (change the hourly cronjob): https://github.com/metabase/metabase/blob/423323e4ae1597499d01f1287763df68c7706927/src/metabase/task/sync_databases.clj

```
;; build our trigger
  (reset! sync-databases-trigger (triggers/build
                                   (triggers/with-identity (triggers/key sync-databases-trigger-key))
                                   (triggers/start-now)
                                   (triggers/with-schedule
                                     ;; run at the end of every hour
                                     (cron/cron-schedule "0 50 * * * ? *"))))
```

**Comment by DannyDouglass** [comment]
@luqmansyauqi thanks for the workaround. I'd rather not create my own build and can work around this in the short term. 

for clarification, I'm asking to control the schedule as I have a sync operation that is populating my reporting data source. I need to ensure metabase isn't trying to sync while I'm restoring my data source.

**Comment by salsakran** [comment]
To elaborate a bit on this, I think there should be a per-db setting that sets

1) When and how often the database is synced (i.e. the schema read and updated)
2) When and how often the database is analyzed (i.e. the fields finger printed, rows counted, etc)

This should have the current settings as a default and only appear in the database details edit screen, not the setup process.

**Comment by camsaul** [comment]
Implemented by #5464

**Comment by mazameli** [comment]
Hey everyone. Just a heads up that [we've shipped version 0.26](http://www.metabase.com/blog/Metabase-0.26) with sync scheduling controls.
